libchisel/src/dropsection.rs

```diff
@@ -3,7 +3,13 @@ use std::error::Error;
 
 use parity_wasm::elements::{Module, Section};
 
-use super::{ChiselModule, ModuleConfig, ModuleError, ModuleKind, ModuleTranslator};
+use super::{ChiselModule, ModuleError, ModuleKind, ModuleTranslator};
+
+impl From<std::num::ParseIntError> for ModuleError {
```

Comment:
Maybe we could just implement `From<E: Error> for ModuleError` so that we can use the ? operator globally. In the end it's all a `Custom` enum value.